### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by bearded-papa** [RandomNinjaAtk/arr-scripts#240] [Q&A] [answered]
How do i go about removing everything installed by the bash script.

I am wanting to stop using this as it is not quite what I need

**Comment by RandomNinjaAtk** [comment, marked as answer]
https://github.com/RandomNinjaAtk/arr-scripts/blob/main/lidarr/readme.md#uninstallationremoval

**Reply by bearded-papa** [reply]
Apologies, I must have missed that when skiming through the readme docs.
Thanks for replying
